lib/matplotlib/mlab.py

```diff
@@ -2867,14 +2867,19 @@ def get_func(name, item, func):
         'print':  'print_',
         }
 
-    def get_converters(reader):
+    def get_converters(reader, comments):
```

Comment:
`comments` is not treated as a list of chars but as a string in the whole `csv2rec` function.